internal/plugintest/working_dir.go

```diff
@@ -135,17 +153,13 @@ func (wd *WorkingDir) ClearPlan() error {
 // Init runs "terraform init" for the given working directory, forcing Terraform
 // to use the current version of the plugin under test.
 func (wd *WorkingDir) Init() error {
-	if _, err := os.Stat(wd.configFilename()); err != nil {
+	if wd.configFilename == "" {
```

Comment:
The original os.Stat() is likely safer here, in case configFilename isn't reset to "". 👍

Reply:
Based on the error message, I feel like the original os.Stat() is here as a test whether SetConfig has ever been called.

I re-added the call to os.Stat().